e2e/src/detail-viewer-role.ts

```diff
@@ -0,0 +1,19 @@
+import { browser, by, element } from 'protractor';
+import {delay} from 'rxjs/operators';
+
+export class DetailViewerRole {
+    navigateTo() {
+        return browser.get(browser.baseUrl += 'detail-viewer/role/1') as Promise<any>;
+    }
+    getTitle() {
+        return element(by.xpath('//detail-viewer/div/div/mat-card')).getText() as Promise<string>;
+    }
+    getJobSummaryText() {
+        element(by.xpath('//*[@id="mat-expansion-panel-header-0"]/span[1]/mat-panel-title')).click();
+        delay(2000);
```

Comment:
Is there any reason a manual delay is here? Are you waiting for an event, or is this some sort of simulation for loading?